src/pages/admin/event/[eventId]/[year]/stats/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Attendee } from "@/components/RegistrationTable/columns";
+import { BasicInformation } from "@/types";
+import { GetServerSideProps } from "next";
+import { useRouter } from "next/router";
+
+type Props = {
+    initialData: BasicInformation[] | null
+}
+
+export default function Statistics() {
+    const router = useRouter()
+
+    return (
+        <main className="bg-primary-color min-h-screen">
+          <div className="container mx-auto p-10 flex flex-col">
+                <span>
+                    <h2 className="text-white">Event Statistics</h2>
+                    <p className="text-baby-blue font-poppins">Statistics {">"} {router.query.eventId} {router.query.year}</p>
+                    {/* 
+                    insert charts here
```

Comment:
Is there going to be a follow-up to this PR soon for integrating all the charts? Otherwise we should make a ticket for this

Reply:
Yes we'll create tickets for this when the backend is connected with the registration table and when login stuff is completed